python/sources/simple-csv-new/main.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+from connector import CsvSourceConnector
+
+# Get the directory of the current script
+script_dir = os.path.dirname(os.path.abspath(__file__))
```

Comment:
This should be done inside the `CsvSourceConnector` class so users don't need to mess with paths too much.

Plus, we can use `pathlib` here:
```
from pathlib import Path

Path('<some_path>').absolute() # gives you the absolute path given the current dir
```

Reply:
In general this is meant to just be an example, they dont have to mess with a path this way when it comes to their own file, really.

This was just meant to make our example easy to execute AFAICT (i just copied the outline from our examples repo).